cloud: update Request Units per Second metric for Premium (#22878)
pingcap/docs: tidb-cloud/releases/tidb-cloud-release-notes.md

Reviewer: ```suggestion

- **TiDB Cloud Premium**

    - Add the `AVG RU/s` metric to the [TiDB Cloud Premium](https://docs.pingcap.com/tidbcloud/premium/?plan=premium) **Metrics** page.
    
        `AVG RU/s` displays the average number of RUs consumed per second over the selected time range, helping you better understand resource consumption.

```

**General changes**

- **TiDB Cloud Premium**

    - Add the `AVG RU/s` metric to the [TiDB Cloud Premium](https://docs.pingcap.com/tidbcloud/premium/?plan=premium) metrics page.
    

>         `AVG RU/s` displays the average RU per second over the selected time range, giving you a clearer view of resource consumption.

Reviewer: ```suggestion
- **TiDB Cloud Premium**

    - Add the `AVG RU/s` metric to the [TiDB Cloud Premium](https://docs.pingcap.com/tidbcloud/premium/?plan=premium) **Metrics** page.
    
        `AVG RU/s` displays the average number of RUs consumed per second over the selected time range, helping you better understand resource consumption.
```

        For more information, see [From Preview to Production: TiDB Cloud Dedicated on Microsoft Azure is Now Generally Available](https://www.pingcap.com/blog/tidb-cloud-dedicated-ga-microsoft-azure/).

- **TiDB Cloud Premium**

    - Add the `AVG RU/s` metric to the [TiDB Cloud Premium](https://docs.pingcap.com/tidbcloud/premium/?plan=premium) metrics page.
    

>         `AVG RU/s` displays the average RU/s over the selected time range, giving you a clearer view of resource consumption.

Author: ```suggestion
        `AVG RU/s` displays the average RU per second over the selected time range, giving you a clearer view of resource consumption.
```

> 

Reviewer: ```suggestion
```
